x/claim/keeper/airdrop_supply.go

```diff
@@ -26,3 +27,34 @@ func (k Keeper) GetAirdropSupply(ctx sdk.Context) (val sdk.Coin, found bool) {
 	k.cdc.MustUnmarshal(b, &val)
 	return val, true
 }
+
+// RemoveAirdropSupply removes the AirdropSupply from the store
+func (k Keeper) RemoveAirdropSupply(ctx sdk.Context) {
+	store := prefix.NewStore(ctx.KVStore(k.storeKey), types.KeyPrefix(types.AirdropSupplyKey))
+	store.Delete([]byte{0})
+}
+
+// InitializeAirdropSupply set the airdrop supply in the store and set the module balance
+func (k Keeper) InitializeAirdropSupply(ctx sdk.Context, airdropSupply sdk.Coin) error {
+	// get the eventual existing balance of the module for the airdrop supply
+	moduleBalance := k.bankKeeper.GetBalance(
+		ctx,
+		k.accountKeeper.GetModuleAddress(types.ModuleName),
+		airdropSupply.Denom,
+	)
+
+	// if the module has an existing balance, we burn the entire balance
+	if moduleBalance.IsPositive() {
```

Comment:
What is the scenario where this will happen?

Reply:
In the case you export and import genesis